### DISCUSSION TITLE
Schedule view for only one day

### DISCUSSION THREAD

**Discussion by eefkrr** [YukiGasai/obsidian-google-calendar#78] [Q&A] [answered]
Is it possible to only display the current day in schedule view?

**Comment by eefkrr** [comment, marked as answer]
Yes, you can set the parameter to 0, meaning no further days will be displayed.

**Comment by Mauwzola** [comment]
I'm not able to limit the display for schedule view to a specific date. The ReadMe doesn't provide any examples so I've tried the following:
date: 2023-08-28
date: '2023-08-28'
date: 20230828

**Reply by Mauwzola** [reply]
I've discovered that the 2023-08-28 format works (as does 2023 08 28) once I add "timeframe: 1". Might want to make the latter the default when a specific date is used. In any case, thanks for creating this PlugIn. Thus far, it's working well for me.
